### PR TITLE
Allow null or empty condition name

### PR DESCRIPTION
**Pull request by brunobord**
## Review

* [x] Tests
* [x] `CHANGELOG.rst` Updated
* [ ] Delete your branch

### CODE REVIEW

File: demo/tests/test_end_point.py [hunk 1]
```diff
@@ -919,6 +919,24 @@ def test_create_form_conditions_invalid_dup(self):
         # now that we allow multiple conditions for one field
         self.assertTrue(serializer.is_valid())
 
+    def test_create_form_conditions_null_name(self):
+        """
+        invalidate the condition using a condition name=None.
```
Comment (syldlb) [inline demo/tests/test_end_point.py]: Are we really "invalidating" the condition here ? I would have said transform / modify

File: demo/tests/test_end_point.py [hunk 2]
```diff
@@ -937,6 +937,24 @@ def test_create_form_conditions_null_name(self):
         self.assertIsNone(condition['name'])
         self.assertIn('action', condition)
 
+    def test_create_form_conditions_empty_name(self):
+        """
+        invalidate the condition using a condition name="".
```
Comment (syldlb) [inline demo/tests/test_end_point.py]: Same as above

**Comment by brunobord** [comment]
oh... this "reset" of the condition name... I'll investigate on it, maybe we can clean this up.

File: formidable/forms/conditions.py [hunk 3]
```diff
@@ -51,11 +53,10 @@ def convert_values(field_id, values):
                                    convert_values(test['field_id'],
                                                   test['values']))
                      for test in condition['tests']]
-            if 'name' not in condition:
```
Comment (syldlb) [inline formidable/forms/conditions.py]: Thanks for the cleanup :+1:

**Comment by brunobord** [comment]
small changes, including in the docstrings (they were nonsense...)

**Pull request merged**